### DISCUSSION TITLE
More Keyboard sounds

### DISCUSSION THREAD

**Discussion by ObiKeahloa** [monkeytypegame/monkeytype#1029] [Ideas] [answered]
Can you please add the sounds of the Anne Pro Keyboard and the Pom jelly Keycaps?
You can get the sounds of both of them from :3ildcat's videos

**Comment by frogfrogfrogfrogfrog** [comment, marked as answer]
Yeah more keyboard sounds would be cool. Especially those keyboards that sound really satisfying to type with.

**Reply by ObiKeahloa** [reply]
Thanks! Is it ok if I reply with a few more? (I can get the sounds of them )

**Reply by ObiKeahloa** [reply]
Is it possible to add more details to the sound settings showing numbers 1-4 , can it show the details about the Keyboard?
I didn't want to open a new discussion just for this

**Reply by frogfrogfrogfrogfrog** [reply]
I'm not a developer of Monkeytype. Just a user that's been helping out for a few months now. I just wanted to reopen this discussion to @Miodec if he wanted to add more keyboard sounds. But a few more keyboard sounds would be cool tbh

**Reply by Miodec** [reply]
Yeah sure. It would be best if you could make a pull request with the new sounds, but if not send some high-quality recordings and ill get around to it when I have free time, or need a break from the important things.

**Reply by ObiKeahloa** [reply]
Ok I will make a pull request.
